tasks/darwin/install.yml

```diff
@@ -15,7 +15,41 @@
   loop_control:
     loop_var: jenkins_swarm_cmd_item
 
-- name: Ensure LaunchDaemon plist file exists
+- name: Set fact for jenkins_swarm_client_legacy_launchd_plist
```

Comment:
I assume we would remove this after everything is deployed?

Reply:
Maybe at some point in the future, but it's a public role, so we don't know the migration schedules of others. Sadly, I think we'll probably need to carry these tasks around for awhile.